talk/expert/cpp20concepts.tex

```diff
@@ -0,0 +1,194 @@
+\subsection[tmpl]{C++20 Requirements and Concepts}
```

Comment:
```suggestion
\subsection[concepts]{C++20 Concepts}
```
The first part (in []) is for the top bar and should be super short, the other part is still in the agenda slide, so should fit in the width of a column

Reply:
ok. I would prefer to keep "Requirements and concepts", dropping C++20 which is already visible thanks to \frametitlecpp[20]. I insist on requirements because I feel most of the complex explanations are about requirements. Concepts are almost a detail, a way to group some requirements.